package.json

```diff
@@ -65,10 +65,11 @@
     "sinon": "1.17.1",
     "stylus": "0.52.4",
     "svgo": "0.5.6",
-    "unexpected": "9.6.0",
-    "unexpected-dom": "2.0.1",
-    "unexpected-mitm": "7.7.0",
-    "unexpected-sinon": "6.4.2",
+    "unexpected": "10.0.1",
+    "unexpected-dom": "3.0.0",
+    "unexpected-messy": "5.0.0",
```

Comment:
Can be omitted (although npm 2 lists it after `npm install --save-dev unexpected-mitm`)